test_scripts/API/Navigation/SendLocation/013_SendLocation_unsupported_resource.lua

```diff
@@ -0,0 +1,81 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/24
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/TRS/embedded_navi/SendLocation_TRS.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- UNSUPPORTED_RESOURCE in case Navi interface is not available on HMI 
+--
+-- Description:
+-- App sends SendLocation will valid parameters but actually Navigation interface is unsupported.
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered on SDL
+
+-- Steps:
+-- appID requests SendLocation with address, longitudeDegrees, latitudeDegrees, deliveryMode and other parameters
+
+-- Expected:
+
+-- SDL validates parameters of the request
+-- SDL checks if Navi interface is available on HMI
+-- SDL checks if SendLocation is allowed by Policies
+-- SDL checks if deliveryMode is allowed by Policies
+-- SDL transfers the request with allowed parameters to HMI
+-- SDL receives response from HMI
+-- SDL transfers response to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSendLocation = require('test_scripts/API/SendLocation/commonSendLocation')
+local hmi_values = require('user_modules/hmi_values')
+
+--[[ Local Variables ]]
+local requestParams = {
+    longitudeDegrees = 1.1,
+    latitudeDegrees = 1.1,
+    addressLines = 
+    { 
+        "line1",
+        "line2",
+    }, 
+    address = {
+        countryName = "countryName",
+        countryCode = "countryName",
+        postalCode = "postalCode"
+    },
+    locationName = "location Name",
+    locationDescription = "location Description",
+    phoneNumber = "phone Number",
+    deliveryMode = "PROMPT"
+}
+
+--[[ Local Functions ]]
+local function disableNavigationInterface(...)
```

Comment:
@Itileda '...' is useless

Reply:
@dboltovskyi , fixed in 6f2e7484bc3d0171c93beefc946839185e141366